build-support/bin/generate_docs.py

```diff
@@ -199,25 +199,41 @@ def create_parser() -> argparse.ArgumentParser:
 
 
 def run_pants_help_all() -> dict[str, Any]:
-    deactivated_backends = [
-        "internal_plugins.releases",
-        "pants.backend.experimental.java.debug_goals",
-        "pants.backend.experimental.scala.debug_goals",
-    ]
-    activated_backends = [
-        "pants.backend.codegen.protobuf.python",
+    # List all (stable enough) backends here.
+    backends = [
         "pants.backend.awslambda.python",
+        "pants.backend.codegen.protobuf.python",
+        "pants.backend.experimental.codegen.thrift.apache.python",
+        "pants.backend.experimental.docker",
+        "pants.backend.experimental.docker.lint.hadolint",
+        "pants.backend.experimental.go",
+        "pants.backend.experimental.java",
+        "pants.backend.experimental.java.lint.google_java_format",
+        "pants.backend.experimental.python.lint.autoflake",
+        "pants.backend.experimental.python.lint.pyupgrade",
+        "pants.backend.experimental.scala",
+        "pants.backend.experimental.scala.lint.scalafmt",
+        "pants.backend.google_cloud_function.python",
+        "pants.backend.plugin_development",
+        "pants.backend.python",
         "pants.backend.python.lint.bandit",
+        "pants.backend.python.lint.black",
+        "pants.backend.python.lint.docformatter",
+        "pants.backend.python.lint.flake8",
+        "pants.backend.python.lint.isort",
         "pants.backend.python.lint.pylint",
         "pants.backend.python.lint.yapf",
+        "pants.backend.python.mixed_interpreter_constraints",
+        "pants.backend.python.typecheck.mypy",
+        "pants.backend.shell",
+        "pants.backend.shell.lint.shellcheck",
+        "pants.backend.shell.lint.shfmt",
     ]
```

Comment:
FYI this is the list of backends. Any you think I should add? I left off these:

* src/python/pants/backend/experimental/codegen/avro/java
* src/python/pants/backend/experimental/codegen/protobuf/{java,scala}
* src/python/pants/backend/experimental/codegen/thrift/apache/java
* src/python/pants/backend/experimental/codegen/thrift/scrooge/{java,scala}
* src/python/pants/backend/experimental/debian
* src/python/pants/backend/experimental/go/lint/vet
* src/python/pants/backend/experimental/java/debug_goals
* src/python/pants/backend/experimental/python
* src/python/pants/backend/experimental/scala/debug_goals
* src/python/pants/backend/experimental/terraform

https://github.com/pantsbuild/pants/blob/c890ec15b341da9e86cbccda4d74cd265f9ba054/src/python/pants/init/BUILD#L9-L49

Reply:
Perhaps some (if not all) stuff in `experimental python` would be nice to include.. ?
Unless they're in such a state that not documenting them is preferable.. ?

Reply:
It looks like that backend is necessary for Twine and `publish` to have automated references, which would be good. I don't love that `user-generate-lockfiles` will also be picked up, but I just finished deprecating that goal, so that's fine with me.

Reply:
cc @tdyas as the author of several of these backends

Reply:
I'd leave out the non-Python codegen backends for now since there is additional work needed to get them actually working.